index.js

```diff
@@ -514,6 +514,56 @@ class GithubScm extends Scm {
         }
     }
 
+    /**
+     * Get a users permissions on an organization
+     * @method getOrgPermissions
+     * @param  {Object}   config                  Configuration
+     * @param  {String}   config.organization     The organization to get permissions on
+     * @param  {String}   config.username         The user to check against
+     * @param  {String}   config.token            The token used to authenticate to the SCM
+     * @param  {String}   [config.scmContext]     The scm context name
+     * @return {Promise}
+     */
+    async _getOrgPermissions(config) {
+        const result = {
+            admin: false,
+            member: false
+        };
+
+        try {
+            const permission = await this.breaker.runCommand({
+                action: 'getOrgMembership',
+                scopeType: 'users',
+                token: config.token,
+                params: {
+                    org: config.organization
+                }
+            });
+            const role = permission.data.role;
+            const state = permission.data.state;
+
+            if (state !== 'active') {
+                return result;
+            }
+
+            result[role] = true;
+
+            return result;
+        } catch (err) {
+            // Suspended user
+            if (err.message.match(/suspend/i)) {
+                winston.info(
+                    `User's account suspended for ${config.scmUri}, ` +
+                    'it will be removed from pipeline admins.');
```

Comment:
will this actually remove it from pipeline admins? or just from org admins?

Reply:
removed that part already, that case should never happen. We are only calling this function when someone tries to create/update/delete a build cluster.